src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -51,28 +51,7 @@ export const getBuildInitiator = (
     }
   });
 
-  let initiator = "";
-  switch (b) {
-    case "gitter_request":
-      initiator = "Commit";
-      break;
-    case "patch_request":
-      initiator = "Patch";
-      break;
-    case "github_pull_request":
-      initiator = "Pull Request";
-      break;
-    case "merge_test":
-      initiator = "Commit Queue";
-      break;
-    case "ad_hoc":
-      initiator = "Periodic Build";
-      break;
-    default:
-      initiator = "";
-  }
-
-  return initiator;
+  return requesterSubscriberOptions[b];
```

Comment:
Since it seems like you wanted to use the empty string if no match is found:
```suggestion
  return requesterSubscriberOptions[b] ?? ";
```

Reply:
thank you!